R/init.R

```diff
@@ -159,10 +163,15 @@ init <- function(data,
   # argument transformations
   ## `modules` - landing module
   landing <- extract_module(modules, "teal_module_landing")
-  landing_module <- NULL
   if (length(landing) == 1L) {
-    landing_module <- landing[[1L]]
+    landing_popup <- landing[[1L]]
     modules <- drop_module(modules, "teal_module_landing")
+    # TODO: verify the version before release.
+    lifecycle::deprecate_soft(
+      when = "0.16",
+      what = "landing_popup_module()",
+      details = "The `landing_popup_module` is supposed to be called as the `landing_popup` argument of the `init` instead of passing it as a module inside the `modules` argument of `init`"
```

Comment:
```suggestion
      details = "Pass `landing_popup_module` to the `landing_popup` argument of the `init` instead of wrapping it into `modules()` and passing to the `modules` argument"
```

or

```r
Pass `landing_popup_module` to the `init(landing_popup)` argument 
instead of wrapping it into `modules()` and passing to the `init(modules)`"
```